main.py

```diff
@@ -56,6 +57,9 @@ def _run(self):
                 self.database.save_submission(submission)
                 continue
 
+            if ExcludedArtists.is_excluded(parsed_submission.artist) is True:
```

Comment:
Can still do the YouTube play count check even if the artist is Various/Anon.  In fact, can still do that check even if we fail to parse *any* information about the song.

Maybe we should reference YouTube directly rather than inside the loop?  Thoughts?

Reply:
not a bad idea actually, will implement this

Reply:
Functionally tested and working: https://www.reddit.com/r/listentothisbottest/comments/fgm05a/adelfeaturinge00hello/